### DISCUSSION TITLE
Adding contidiotioned HTTP header

### DISCUSSION THREAD

**Discussion by taxmeifyoucan** [nginx-proxy/nginx-proxy#2034] [Q&A] [answered]
I am migrating from manual nginx setup to `nginx-proxy` docker and I have issue setting up correct configurtion. I see it's possible to modify `/etc/nginx/proxy.conf` but I am not sure if that is sufficient to copy my current config from `/etc/nginx/sites-enabled/default `. In my current config, I am adding headers based on HTTP methond like so: 
```
	location / {
  if ($request_method = 'OPTIONS') {
      add_header 'Access-Control-Allow-Origin' '*';
      add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS';
      return 200;
    }
    if ($request_method = 'GET' ) {
      add_header 'Access-Control-Allow-Origin' '*';
      add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS';
      return 200;
    }
    if ($request_method = 'POST') {
      add_header 'Access-Control-Allow-Origin' '*';
    }
    proxy_http_version 1.1;
    proxy_set_header Host $host;
    proxy_pass http://localhost:8000
  }
```

Is there a way how to create this kind of config with nginx-proxy container? Thank you folks!

**Comment by kvbik** [comment, marked as answer]
an answer to this one is actually pretty straight-forward and covered in the documentation [1]

[1] https://github.com/nginx-proxy/nginx-proxy/blob/main/README.md#per-virtual_host-location-default-configuration

see [2] to get an inspiration

[2] https://github.com/taxmeifyoucan/balancer/commit/6184558022c89f928f353cdcc3e8488eb7f0b83b

thx!
